docker/manage

```diff
@@ -201,33 +163,28 @@ configureEnvironment() {
   # controller
   export CONTROLLER_SERVICE_PORT=5201
   export CONTROLLER_URL="${CONTROLLER_URL:-http://controller:5000}"
-  export CONTROLLER_NGROK="http://controller-ngrok:4040"
   export CONTROLLER_WEB_HOOK_URL=${CONTROLLER_WEB_HOOK_URL:-${CONTROLLER_URL}/webhooks}
   if [ ! -z "${CONTROLLER_API_KEY}" ]; then
-    CONTROLLER_WEB_HOOK_URL="${CONTROLLER_WEB_HOOK_URL}/${CONTROLLER_API_KEY}"
+    CONTROLLER_WEB_HOOK_URL="${CONTROLLER_WEB_HOOK_URL}#${CONTROLLER_API_KEY}"
   fi
-
-  # Multi-Tenant ACA-Py config
-  export ACAPY_HOST="http://tenant-proxy"
-  export ACAPY_TENANCY="multi"
-  export MT_ACAPY_WALLET_ID="ece83a72-81df-40b0-b770-9a55d92b254d"
-  export ACAPY_NGROK_TUNNEL_HOST="http://ngrok-traction-agent:4040"
-  export AGENT_ADMIN_PORT="8080"
-
-
-  # # # Single-Tenant ACA-Py config
-  # export ACAPY_HOST="http://aca-py"
-  # export ACAPY_TENANCY="single"
-  # export ST_ACAPY_ADMIN_API_KEY_NAME="x-api-key"
-  # export ST_ACAPY_ADMIN_API_KEY="change-me"
-  # export ACAPY_NGROK_TUNNEL_HOST="http://aca-py-ngrok:4040"
-  # export AGENT_ADMIN_PORT="8077"
-
-  # ACA-py url constructions
-  export AGENT_HTTP_PORT="8030"
-  export ACAPY_ADMIN_URL="${ACAPY_HOST}:${AGENT_ADMIN_PORT}"
-  export AGENT_ENDPOINT=${AGENT_ENDPOINT:-http://$DOCKERHOST:$AGENT_HTTP_PORT}
-  export ACAPY_AGENT_URL="${AGENT_ENDPOINT:-${ACAPY_HOST}:$AGENT_HTTP_PORT}"
+  export ST_ACAPY_ADMIN_API_KEY_NAME="x-api-key"
+
+  # agent
+  export AGENT_TENANT_MODE="${AGENT_TENANT_MODE:-single}"
+  export AGENT_HOST="http://aca-py"
+  export AGENT_NAME="VC-AuthN Agent"
+  export AGENT_HTTP_PORT=${AGENT_HTTP_PORT:-8030}
+  export AGENT_ADMIN_PORT=${AGENT_ADMIN_PORT:-"8077"}
+  export AGENT_ADMIN_URL=${AGENT_ADMIN_URL:-http://$AGENT_HOST:$AGENT_ADMIN_PORT}
+  export AGENT_ENDPOINT=${AGENT_ENDPOINT:-http://$AGENT_HOST:$AGENT_HTTP_PORT}
```

Comment:
AGENT_HOST is set later in the options, so this gets prefixed with `http://aca-py` even when AGENT_HOST is set to `http://<traction-acapy-host>` later.

Reply:
The latest commit should fix this issue.